testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-delta-lake-databricks/tempto-configuration.yaml

```diff
@@ -7,7 +7,7 @@ databases:
     prepare_statement:
       - USE ${databases.delta.schema}
     table_manager_type: jdbc
-    jdbc_url: ${DATABRICKS_JDBC_URL}
+    jdbc_url: ${DATABRICKS_JDBC_URL};EnableArrow=0
```

Comment:
Please update the `DATABRICKS_JDBC_URL` secret instead.
We have now proof that it works with `Arrow` disabled, so we can safely update the secret.

Reply:
I disagree with putting the option in secret. People who can see `DATABRICKS_JDBC_URL` value is limited.

Reply:
This is true. Should I make eventually a new PR which exposes the full jdbc URL and hides only the Databricks components:
- organisation
- account
- cluster

?